packages/main/src/mainWindow.ts

```diff
@@ -234,3 +238,11 @@ export async function restoreWindow() {
     window.focus();
   }
 }
+
+// Checks process args if it was started with the --minimize flag
+function ifStartedMinimize(): boolean {
+  // We convert to string only because sometimes [node] will be the first argument in a packaged
+  // environment, so instead of checking each element, simply convert to string and see if --minimize was included.
+  const argv = process.argv.toString();
```

Comment:
shouldn't we use `app.commandLine.hasSwitch` ?

Reply:
Unfortunatley does not work on Windows / does not detect the --minimized parameter, but `process.argv` does. So ended up using process.argv instead. You can try it by adding `  console.log(app.commandLine.hasSwitch('minimize'));` and you'll see.